pkg/db/db.go

```diff
@@ -561,15 +597,19 @@ func (h *DBHandler) processReleaseRow(ctx context.Context, err error, rows *sql.
 			return nil, fmt.Errorf("Error during json unmarshal of manifests for releases. Error: %w. Data: %s\n", err, metadataStr)
 		}
 		row.Manifests = manifestData
-
-	} else {
-		row = nil
+		result = append(result, row)
+	}
+	for i := 0; i < len(result); i++ {
+		if result[i].EslId != maxEslId {
+			result = append(result[:i], result[i+1:]...)
```

Comment:
This would have to mean that all of the most recent changes to all releases from some application are in sync regarding the eslID. For example, the query returns:

_row 1: eslID = 1, appName = app, releaseVersion = 1
row 2: eslID = 2, appName = app, releaseVersion = 2
row 3: eslID = 1, appName = app, releaseVersion = 3

Following this implementation, we would only return row2, while we would want to return all 3 rows, as they are the most recent changes to each version, right? 
I think this scenario is possible

Reply:
And there might be 100 releases that need to be changed, not just the latest one added to the table. You need to change every single active release that is stored in the all_releases table